share/patch_mysql_db_zonemaster_backend_ver_5.0.2.pl

```diff
@@ -0,0 +1,26 @@
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
+use Encode;
+
+use DBI qw(:utils);
+
+use Zonemaster::Backend::Config;
+
+die "The configuration file does not contain the MySQL backend" unless (lc(Zonemaster::Backend::Config->load_config()->BackendDBType()) eq 'mysql');
```

Comment:
BackendDBType() returns the exact string `'MySQL'` so it's better to compare against that.

Also, consider adding a newline before `unless`.

Reply:
I just copied the "5.0.0" script and changed as little as possible. Wouldn't it be better to do that for next release in all five scripts? I can create an issue and do that for next release.

Reply:
Issue #618 created.